src/k2/cmd/tpcc/tpcc_client.cpp

```diff
@@ -149,6 +151,23 @@ class Client {
     }
 
 private:
+    std::vector<uint32_t> _parse_weights(k2::String txn_weights) {
```

Comment:
I don't think you need a custom parser for this. You can just define the option in config as:
```c++
("txn_weights", bpo::value<std::vector<int>>()->multitoken()->default_value(std::vector<int>()), "A space-delimited list of exactly 5 elements denoting the percentage for each txn type:" <list txn types here>)
```

and then pull it out with a configVar:
```c++
    ConfigVar<std::vector<int>> _txn_weights{"txn_weights"};
```

Reply:
nice, didn't know this feature. will do.

Reply:
done